apps/avifenc.c

```diff
@@ -1044,18 +1099,18 @@ int main(int argc, char * argv[])
         usingAOM = AVIF_TRUE;
     }
     avifBool hasAlpha = (image->alphaPlane && image->alphaRowBytes);
-    avifBool losslessColorQP = (minQuantizer == AVIF_QUANTIZER_LOSSLESS) && (maxQuantizer == AVIF_QUANTIZER_LOSSLESS);
-    avifBool losslessAlphaQP = (minQuantizerAlpha == AVIF_QUANTIZER_LOSSLESS) && (maxQuantizerAlpha == AVIF_QUANTIZER_LOSSLESS);
+    avifBool losslessColorQuality = (quality == AVIF_QUALITY_LOSSLESS);
```

Comment:
maybe rename to: usingLosslessColor or hasLosslessColor to be consistent with the naming around.

Reply:
Done. Changed the variable names to `usingLosslessColor` and `usingLosslessAlpha`. Ideally the variable names should contain the word "quality", but the names would be too verbose.